react/src/components/organisms/UtilityNav/index.js

```diff
@@ -118,13 +120,14 @@ const NavItem = (obj) => {
 const NavItemLink = (obj) => {
   const item = obj.data;
   const iconProps = {
-    name: item.icon,
     ariaHidden: true
   };
+  const IconComponent = item.icon === 'building' ? Building : Login;
```

Comment:
not sure why we are only allow these two icon options.. we should make this a rendered prop to pass in whatever Icon, but this can be in a separate PR.